tests/test_admin.py

```diff
@@ -296,6 +294,14 @@ def test_content_link(self):
             ),
         )
 
+
+class VersionAdminViewTestCase(CMSTestCase):
+
+    def setUp(self):
+        self.superuser = self.get_superuser()
+        versioning_extension = apps.get_app_config('djangocms_versioning').cms_extension
+        self.versionable = versioning_extension.versionables_by_content[PollContent]
```

Comment:
As previously mentioned, I think it's better to just import the versionable from polls/cms_config.py rather than rely on this to set up data.

Reply:
Here's an example: https://github.com/divio/djangocms-versioning/blob/master/tests/test_cms_config.py#L128 This avoids set up having a dependency on our code and only depends on test_utils which is fine.